Kitodo/src/main/java/org/goobi/mq/processors/CreateNewProcessProcessor.java

```diff
@@ -211,22 +213,19 @@ private static Template getTemplateByTitle(String templateTitle) {
     /**
      * The function docTypeIsPossible() tests whether a given docType String can
      * be applied to a given process template. If so, it will return “true”,
-     * otherwise, it will throw an informative IllegalArgumentException.
+     * otherwise false.
      *
      * @param dialog
      *            the ProzesskopieForm object to test against
      * @param docType
      *            the desired docType ID string
      * @return true on success
-     * @throws IllegalArgumentException
-     *             if a docType is not applicable to the template or the docType
-     *             isn’t valid
      */
     private static boolean docTypeIsPossible(ProzesskopieForm dialog, String docType) {
         Boolean fieldIsUsed = dialog.getStandardFields().get("doctype");
         if (fieldIsUsed == null || fieldIsUsed.equals(Boolean.FALSE)) {
-            throw new IllegalArgumentException(
-                    "Bad argument “docType”: Selected template doesn’t provide " + "the standard field “doctype”.");
+            logger.warn("Bad argument “docType”: Selected template doesn’t provide the standard field “doctype”.");
```

Comment:
- Did you checked if return value of this method is used in calling methods? 
- Did you checked if previous thrown exception is handled in calling methods?

Reply:
Yes, it is used:

```java
if (docType != null && docTypeIsPossible(newProcess, docType)) {
```

But now when I take a second look at this both methods I have still feeling that something is not right with them...